### PR TITLE
Implemented Array#sort() (#24)

### PR DESCRIPTION
**Pull request by j-blandford**
Also has the optional comparator

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -470,6 +470,18 @@ class Collection {
...
+  /**
+   * Returns a sorted list of all collection items, with an optional comparator
+   *
+   * @param {Function} comparator
+   *
+   * @returns {Collection}
+   */
+  async sort (comparator) {
+    this.items = [...this.items.sort(comparator)]
+    return this
```
Comment (marcuspoehls) [inline src/collection.js:483]: Can you please add an empty line before the return statement?

**Review by marcuspoehls — CHANGES_REQUESTED** [review]
@JamesBlandford Hey James, thank you for the PR! Looks good and I'm just requesting a minor formatting change :) Can you please have a look?

File: src/collection.js (hunk 1 above)
Comment (j-blandford) [reply src/collection.js:483]: just pushed the formatting change

**Review by marcuspoehls — APPROVED** [review]

**Comment by marcuspoehls** [comment]
@JamesBlandford Thank you James! Merged :rocket:

**Pull request merged**